### DISCUSSION TITLE
To create an issue

### DISCUSSION THREAD

**Discussion by rammya29** [prathimacode-hub/Awesome_Python_Scripts#375] [Ideas]
I want to create an issue for Number Plate Recognition. Can I do that?

**Comment by prathimacode-hub** [comment]
First read the README and CONTRIBUTING GUIDELINES of the project. See the kind of scripts this repo is aceepting. Then think of some related issue you want to work on. Check if the issue you want to contribute is already available in the repo or got assigned. Then raise an issue using the template given in README FILE of the project. For reference, you can check up with other assigned issues to know how template looks like. There is a procedure to follow. Kindly do that. All the best. Any queries, don't hesitate to ask. @rammya29
